18/README.md

```diff
@@ -42,22 +42,36 @@ A Validator must be able to discern valid Votes from invalid ones. A valid vote
 ### What is a valid Election?
 A Validator must be able to discern valid Elections from invalid ones. A valid Election is a transaction where **all** the following conditions are true.
 
-1. `operation` is `CREATE`, or equivalent.
-1. The `inputs` satisfy the conditions for a CREATE transaction. For example, the public key listed in `inputs.owners_before` is the public key of the Initiator.
+1. The transaction complies to the `CREATE` transaction JSON schema.
+1. The transaction additionally complies to the [election JSON schema](#./election_transaction.yaml).
```

Comment:
I cannot seem to find `./election_transaction.yaml`

Reply:
My bad, I forgot to add it.